### PR TITLE
Doc  Improvements

### PR DESCRIPTION
**Pull request by shanaxel42**

### CODE REVIEW

File: starfish/codebook/codebook.py [hunk 1]
```diff
@@ -546,10 +546,10 @@ def metric_decode(
         metric : str
             the sklearn metric string to pass to NearestNeighbors
 
-        See Also
-        --------
-        The available norms for this function can be found at the following link:
-        https://docs.scipy.org/doc/numpy-1.14.0/reference/generated/numpy.linalg.norm.html
+        Notes
+        ------
+        The available norms for this function can be found `here
```
Comment (neuromusic) [inline starfish/codebook/codebook.py]: is it worth setting up intersphinx? http://www.sphinx-doc.org/en/master/usage/extensions/intersphinx.html

Comment (shanaxel42) [reply starfish/codebook/codebook.py]: I did set up intersphinx! https://github.com/spacetx/starfish/commit/1153bc42f561c4bdccd500e2847e94c89836a98a#diff-1ec61223727a10e5e4c74c68336809fcR58  But just have xarray in the mappings right now, we can add numpy too if we want

Comment (neuromusic) [reply starfish/codebook/codebook.py]: oh cool!! yeah, if we're going to link out to NumPy docs, might as well use intersphinx to do it

Comment (shanaxel42) [reply starfish/codebook/codebook.py]: sweet I'm on it

File: starfish/codebook/codebook.py [hunk 2]
```diff
@@ -514,9 +510,11 @@ def decode_metric(
             norm_order: int, metric: str='euclidean'
     ) -> IntensityTable:
         """
-        Assigns intensity patterns that have been extracted from an :py:class:`ImageStack` and
-        stored in an :py:class:`IntensityTable` by a :py:class:`SpotFinder` to the gene targets that
-        they encode.
+        Assigns intensity patterns that have been extracted from an
+        :py:class:`~starfish.imagestack.imagestack.ImageStack` and
+        stored in an :py:class:`~starfish.intensity_table.intensity_table.IntensityTable` by a `SpotFinder
+        <https://spacetx-starfish.readthedocs.io/en/latest/api/spots/detection/detection.html>`_
```
Comment (neuromusic) [inline starfish/codebook/codebook.py]: a sphinx cross-reference here will be more robust in the event of changing urls, rendering in PDF, or a re-org of the docs https://www.sphinx-doc.org/en/1.7/markup/inline.html#cross-referencing-arbitrary-locations

File: starfish/codebook/codebook.py [hunk 3]
```diff
@@ -580,9 +578,11 @@ def decode_metric(
 
     def decode_per_round_max(self, intensities: IntensityTable) -> IntensityTable:
         """
-        Assigns intensity patterns that have been extracted from an :py:class:`ImageStack` and
-        stored in an :py:class:`IntensityTable` by a :py:class:`SpotFinder` to the gene targets that
-        they encode.
+        Assigns intensity patterns that have been extracted from an
+        :py:class:`~starfish.imagestack.imagestack.ImageStack` and
+        stored in an :py:class:`~starfish.intensity_table.intensity_table.IntensityTable` by a `SpotFinder
+        <https://spacetx-starfish.readthedocs.io/en/latest/api/spots/detection/detection.html>`_
```
Comment (neuromusic) [inline starfish/codebook/codebook.py]: cross-ref

File: starfish/image/_segmentation/watershed.py [hunk 4]
```diff
@@ -16,35 +16,35 @@
...
+              threshold to apply to nuclei image
+          input_threshold : Number
+              threshold to apply to stain image
+          min_distance : int
+              minimum distance before object centers in a provided nuclei image are considered single
+              nuclei
+
+          Notes
+          ------
+          See Also `Watershed <https://scikit-image.org/docs/dev/auto_examples/segmentation/plot_watershed.html>`_
```
Comment (neuromusic) [inline starfish/image/_segmentation/watershed.py]: intersphinx to skimage?

File: starfish/spots/_detector/blob.py [hunk 5]
```diff
@@ -66,9 +66,9 @@ def __init__(
         is defined as an absolute value -- therefore it must be adjusted depending on the datatype
         of the passed image.
 
-        See Also
-        --------
-        http://scikit-image.org/docs/dev/auto_examples/features_detection/plot_blob.html
+        Notes
+        -----
+        See Also `<http://scikit-image.org/docs/dev/auto_examples/features_detection/plot_blob.html>`_
```
Comment (neuromusic) [inline starfish/spots/_detector/blob.py]: intersphinx?

**Pull request closed without merging**